apps/engmanagement/src/pages/[slug].tsx

```diff
@@ -0,0 +1,153 @@
+import * as React from 'react'
+import {GetServerSideProps} from 'next'
+import {serialize} from 'next-mdx-remote/serialize'
+import {MDXRemote} from 'next-mdx-remote'
+import isEmpty from 'lodash/isEmpty'
+import find from 'lodash/find'
+import ArticleTemplate from 'templates/article'
+import ConvertkitSubscribeAndTagForm from 'components/convertkit-subscribe-and-tag'
+import groq from 'groq'
+import {sanityClient} from 'utils/sanity-client'
+import checkSubscriber from 'utils/check-subscriber'
+import {useNextSanityImage} from 'next-sanity-image'
+import Image from 'next/image'
+import {useConvertkit} from '@skillrecordings/convertkit'
+import {TheFutureOfRemoteWorkBackground} from 'components/backgrounds'
+
+export default function ExampleChapter({post, source, authorized}: any) {
+  const {
+    title,
+    slug,
+    mainImage,
+    description,
+    ckTagId,
+    publishedAt,
+    ogImage,
+    subscribersOnly,
+  } = post
+
+  const {subscriber} = useConvertkit()
+
+  const MainImage = () => {
+    const imageProps: any = useNextSanityImage(sanityClient, mainImage)
+    if (!mainImage) return null
+    return (
+      <Image
+        src={imageProps.src}
+        {...imageProps}
+        alt=""
+        layout="fill"
+        className="md:object-contain object-cover object-top select-none pointer-events-none"
+        quality={100}
+        placeholder="blur"
+        loading="eager"
+        priority
+      />
+    )
+  }
+  return (
+    <div className="bg-[#111725]">
```

Comment:
this hex color being hard coded like this might benefit from extracting the color into a named `const` so it isn't as confusing. I'd consider passing it as a prop too.

Reply:
donezo 👍